pkg/async/notifications/implementations/sendgrid_emailer_test.go

```diff
@@ -41,3 +45,54 @@ func TestCreateEmailer(t *testing.T) {
 	emailer := NewSendGridEmailer(cfg, promutils.NewTestScope())
 	assert.NotNil(t, emailer)
 }
+
+func TestReadAPIFromEnv(t *testing.T) {
+	envVar := "sendgrid_api_test_key"
```

Comment:
not sure if it's worth storing what env var (if any) was here originally and resetting after the test runs